templates/pages/core/get-started/installation-medias.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Create installation medias for Ubuntu Core
+description: These steps will walk you through creating a bootable Ubuntu Core SD Card or USB flash drive.
+---
+
+# Create installation medias for Ubuntu Core
+
+These steps will walk you through creating a bootable Ubuntu Core SD Card or USB flash drive.
+
+This will erase any existing content on the removable drive.
+
+The process is the same on any OS, you will: download an image file, insert your removable drive (SD Card, USB flash drive, etc.), unmount it, replace the drive content with the image file content, eject the drive.
+
+You can follow the detailed steps:
+
+* [On Ubuntu &rsaquo;](#on-ubuntu)
+* [On Windows &rsaquo;](#on-windows)
+* [On Mac OS &rsaquo;](#on-mac-os)
+
+## On Ubuntu
+
+ 1. Download the Ubuntu Core image for your device in your `Downloads` folder
```

Comment:
It's ordered, but in MarkDown, you can start with a 1. then continue the list with *, it will be rendered as an ordered list